bindings/python/src/session.cpp

```diff
@@ -791,6 +792,11 @@ void bind_session()
     }
 
     {
+        scope s = class_<dummy12>("reopen_network_flags_t");
+        s.attr("reopen_map_ports") = lt::session::reopen_map_ports;
```

Comment:
please add this as an attribute directly on the ``session`` class. this hackery with dummy classes is something I've done mostly to stay API compatible when converting enums into type-safe constants.

Reply:
OK